src/markdown-pages/terraform/terragrunt-configuration.mdx

```diff
@@ -0,0 +1,600 @@
+
+Start by initializing a working directory:
+
+You will also need to install Terragrunt to follow this guide:
+```bash
+brew install terragrunt
+```
+
+To follow the examples in this guide, the accompanying example code is available on [GitHub](https://github.com/jsbnr/nr-terragrunt-intro-example)
+
+```bash
+git clone https://github.com/jsbnr/nr-terraform-intro-example.git
+```
+
+
+<Steps>
+
+<!-- TTERRAGRUNT CONFIGURATION -->
+
+<Step>
+
+## Create a new configuration
+
+Terragrunt is a wrapper that provides extra tooling for your Terraform configurations. In the next steps, you will create a new configuration using Terragrunt, Terraform, and New Relic.
+
+Start by initializing a working directory:
+```bash
+mkdir terragrunt-config && cd terragrunt-config
+```
+In the new directory, create a `terragrunt.hcl` file, create a `./src` directory, and create a `./environments` directory:
+```bash
+touch terragrunt.hcl
+mkdir src
+mkdir environments
+```
+Inside of your `environments` folder, create a folder named `dev`.
+In the `./src` directory, create a `main.tf` and `provider.tf` files:
+```bash
+mkdir environments/dev
+touch src/main.tf
+touch src/provider.tf
+```
+
+In the __provider.tf__ file add the appropriate Terraform and New Relic provider configuration information:
+```hcl
+terraform {
+	required_version = "~> 0.14.3"
+	required_providers {
+		newrelic = {
+			source = "newrelic/newrelic"
+			version = "~> 2.14.0"
+		}
+	}
+}
+```
+
+In the __main.tf__ add a New Relic alert policy named DemoPolicy:
+```hcl
+resource "newrelic_alert_policy" "DemoPolicy" {
+	name = "My Demo Policy"
+}
+```
+
+In the `./environments/dev` directory create a new file named `terragrunt.hcl`
+```bash
+cd ./environments/dev/
+touch terragrunt.hcl
+```
+
+Within the dev folder update the `terragrunt.hcl` file by adding the following include statement. The include statement tells Terragrunt to use any `.hcl` confgiuration files it finds in parent folders.
+```hcl
+include {
+	path= find_in_parent_folders()
+}
+```
+
+Next, add a Terraform block to provide Terragrunt with a Terraform source reference.
+```hcl
+terraform {
+	source = "../../src"
+}
+```
+
+Now you need to configure the provider with your New Relic account details. You will need your [New Relic user API key](https://one.newrelic.com/launcher/api-keys-ui.api-keys-launcher) and account ID.
+```hcl
+provider "newrelic" {
+	account_id = 12345 # Your New Relic account ID
+	api_key = "NRAK-***" # Your New Relic user key
+	region = "US" # US or EU (defaults to US)
+}
+```
+
+<Callout variant="tip">
+
+If you have a single account: The account ID is displayed on the API keys page.
```

Comment:
Is there a link we can provide to the API keys page to make it easy for the reader to visit?